libcontainer/configs/cgroup_unix.go

```diff
@@ -120,5 +120,5 @@ type Resources struct {
 	NetPrioIfpriomap []*IfPrioMap `json:"net_prio_ifpriomap"`
 
 	// Set class identifier for container's network packets
-	NetClsClassid string `json:"net_cls_classid"`
+	NetClsClassid *uint32 `json:"net_cls_classid"`
```

Comment:
Okay, looking at the kernel docs the default `classid` is `0`. So IMO we can just use `!= 0` for `Set()` to decide whether or not to write the value.